### PR TITLE
bones

### PR DESCRIPTION
**Pull request by Deniied**
do not summon davey's mother at 3 am (bones)

### CODE REVIEW

**Comment by joritochip** [comment]
no

**Comment by DaEnder** [comment]
ok

File: MainModule/Server/Commands/HeadAdmins.lua [hunk 1]
```diff
@@ -446,5 +446,25 @@ return function(Vargs, env)
...
+				while wait() do
+					local bone = Instance.new("Part",workspace)
+					bone.Name = "bone"
+					bone.Size = Vector3.new(0.35,0.53,4.36)
+					local boneMesh = Instance.new("SpecialMesh",bone)
+					boneMesh.MeshId = "rbxassetid://523610338"
+					boneMesh.Scale = Vector3.new(0.025, 0.025, 0.025)
+				end
+			end;
+		};
```
Comment (joritochip) [inline MainModule/Server/Commands/HeadAdmins.lua:468]: Remove this part

**Review by joritochip — CHANGES_REQUESTED** [review]

File: MainModule/Server/Commands/HeadAdmins.lua (hunk 1 above)
Comment (Deniied) [reply MainModule/Server/Commands/HeadAdmins.lua:468]: bones

File: MainModule/Server/Commands/HeadAdmins.lua [hunk 2]
```diff
@@ -446,5 +446,25 @@ return function(Vargs, env)
...
+		bones = {
+			Prefix = Settings.Prefix;
+			Commands = {"bones"};
+			Args = {};
+			Description = "bones (very destruct)";
+			AdminLevel = "HeadAdmins";
+			Filter = true;
+			IsCrossServer = false;
+			Function = function(plr,args)
+				while wait() do
```
Comment (ccuser44) [inline MainModule/Server/Commands/HeadAdmins.lua:459]: maybe a while loop is too agressive

You should also add a function to stop these from spawning (or ideally make it only spawn a certain amount)

File: MainModule/Server/Commands/HeadAdmins.lua [hunk 3]
```diff
@@ -446,5 +446,25 @@ return function(Vargs, env)
...
+					local bone = Instance.new("Part",workspace)
```
Comment (ccuser44) [inline MainModule/Server/Commands/HeadAdmins.lua:460]: You should put table.insert(Variables.Objects, bone)

So adonis can automatically clear these

Comment (Expertcoderz) [reply MainModule/Server/Commands/HeadAdmins.lua:460]: Yeah~~, well actually no we want the bones to fill up the server and squeeze em players to dea—~~

**Comment by Expertcoderz** [comment]
wait this PR is giving me inspiration—

**Comment by Deniied** [comment]
> 
> 
> wait this PR is giving me inspiration—

I just wanted to add bones man, this guy named joritochip disrespected davey's mother......

**Pull request closed without merging**